runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ValidateRunnerXlangTest.java

```diff
@@ -110,6 +137,20 @@ private void waitForReady() {
     }
   }
 
+  /**
+   * Motivation behind <i>singleInputOutputTest</i>.
+   *
+   * <ul>
```

Comment:
Not sure whether we should use markup here since these comments will be mostly read from the source code, not from the formatted doc pages (this class doesn't create APIs, it's only for tests). Markups could hinder readability in the raw form.

Reply:
Makes sense. I've removed markup from both files making it more human-readable. Thanks for the advice @ihji!